### ISSUE TITLE
Emacross in DCA doesnt show true

### ISSUE THREAD

**Issue by jokaorgua** [taniman/profit-trailer#1568]
When the coin is in DCA and sell strategy is emacross there is no green 'true' sign if emacross is -0.03 and trigger is -0.01

`DEFAULT_DCA_A_buy_strategy = EMACROSS

DEFAULT_DCA_A_buy_value = -0.01

DEFAULT_DCA_A_buy_value_limit = -12`

**Comment by taniman** [comment]
well did the cross just happened withing the last x candles?

**Comment by taniman** [comment]
ema_cross_candles

**Comment by ofilipowicz** [comment]
I experienced similar issue. I actually spent like 2h trying to find out what is wrong with my settings and why EMACROSS doesn't work as expected. I used indicator from TradingView with my settings. There are multiple coins where cross did happen within given range but EMACROSS and all buy values were met (set it to -0.01) is still false. I noticed it turns green the moment cross happens or exactly after ema_cross_candles. It does it for 1 timeframe and then turns red.
